src/three-components/RoughnessMipmapper.ts

```diff
@@ -0,0 +1,181 @@
+/* @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LinearMipMapLinearFilter, Mesh, MeshStandardMaterial, NoBlending, OrthographicCamera, PlaneBufferGeometry, RawShaderMaterial, Scene, Vector2, WebGLRenderTarget} from 'three';
+import {_Math} from 'three/src/math/Math';
+
+import {sceneRenderer} from './Renderer';
+import {roughness2variance, variance2roughness, varianceDefines} from './shader-chunk/common.glsl';
```

Comment:
`roughnessToVariance`, `varianceToRoughness` (unless this is copying some upstream convention from Three.js)